cmd/ipfs/kubo/add_migrations.go

```diff
@@ -83,10 +83,12 @@ func addMigrationFiles(ctx context.Context, node *core.IpfsNode, paths []string,
 
 		fi, err := f.Stat()
 		if err != nil {
+			f.Close()
 			return err
 		}
 
-		ipfsPath, err := ufs.Add(ctx, files.NewReaderStatFile(f, fi), options.Unixfs.Pin(pin, ""))
+		ipfsPath, err := ufs.Add(ctx, files.NewReaderStatFile(f, fi), options.Unixfs.Pin(pin))
```

Comment:
I am not sure how this is supposed to work without the 2nd argument to [`options.Unixfs.Pin`](https://github.com/ipfs/kubo/blob/master/core/coreiface/options/unixfs.go#L285). Is there a missing change? 

Try running `go test` in this directory on your local machine.